Makefile

```diff
@@ -113,7 +113,7 @@ get-test-namespace: ./out/test-namespace
 
 # E2E test
 .PHONY: e2e-setup
-e2e-setup: e2e-cleanup
+e2e-setup: ./vendor e2e-cleanup
```

Comment:
Vendor directory is already committed to git. So this is not required.

Reply:
Ok, I'll remove it.